python/cudf/cudf/tests/test_series.py

```diff
@@ -572,6 +572,18 @@ def test_series_value_counts(dropna, normalize):
         assert_eq(expect, got, check_dtype=False, check_index_type=False)
 
 
+@pytest.mark.parametrize("bins", [1, 2, 3])
+@pytest.mark.parametrize("dropna", [True, False])
+def test_series_value_counts_bins(dropna, bins):
+    psr = pd.Series([1.0, 2.0, 2.0, 3.0, 3.0, 3.0])
```

Comment:
This test is not having `<NA>` values to really validate `dropna=False/True`

Reply:
Ah, must have removed the `<NA> ` values by mistake. Let me test in now